src/nimlsp.nim

```diff
@@ -145,9 +147,20 @@ proc getProjectFile(file: string): string =
       fileExists(path / current.addFileExt(".nims"))) and certainty <= Cfg:
       result = path / current.addFileExt(".nim")
       certainty = Cfg
-    if fileExists(path / current.addFileExt(".nimble")) and certainty <= Nimble:
-      # Read the .nimble file and find the project file
-      discard
+    for file in walkDirRec(path):
+      if file.endsWith(".nimble") and certainty <= Nimble:
```

Comment:
Maybe this should prefer `current.addFileExt(".nimble")` over `file.endsWith(".nimble")` in case there is more than one?

Reply:
assuming project file base name same as nimble for now.

Reply:
Yeah that's fine. But if two .nimble files are found beside each other in the same folder this would arbitrarily pick one even though one of them is named the same as the current file but with the `.nimble` extension.